### DISCUSSION TITLE
Window switcher style Windows 10 shows Windows NT window switcher

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#547] [General]
Using  22000.348.40.7 and Win 11 22000.348
Selecting Window switcher style Windows 10 ( and restarting File Explorer from EP Properties GUI) shows Windows NT window switcher.
Selecting SWS, Windows NT or Windows 11 style work okay.
I have not uninstalled or even restarted PC to try to fix it, just reporting it here in case it is easily replicated.

Edit: It is caused by restarting File Explorer from EP Properties GUI when switching from Windows 11 style to Windows 10 style.
If I switch from 11 to 10 style but do not restart File Explorer it works.
But the * implies that File Explorer needs to be restarted when changing any Window Switcher style.

**Comment by valinet** [comment]
Confirmed. Created https://github.com/valinet/ExplorerPatcher/issues/548
